cloudify_cli/commands/secrets.py

```diff
@@ -96,11 +102,25 @@ def create(key,
         raise CloudifyCliError('Failed to create secret key. '
                                'Missing option '
                                '--secret-string or secret-file.')
+
+    # Set the secret's schema [default: {"type": "string"}]
+    try:
+        schema = json.loads(secret_schema)
+    except json.decoder.JSONDecodeError as e:
+        raise CloudifyCliError(
+            f'Error decoding JSON schema "{schema}: {e.args[0]}')
```

Comment:
```suggestion
            f'Error decoding JSON schema "{schema}: {e}')
```
tbh, wasn't `Exception.args` deprecated some time ago? but anyway just `str`'ing the exception like this seems to work nice:
```python
>>> x.args
('Expecting value: line 1 column 1 (char 0)',)
>>> str(x)  
'Expecting value: line 1 column 1 (char 0)'
```